crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -58,22 +59,39 @@ pub(crate) fn create(
     // considered the "base" for the virtual environment. This is typically the Python executable
     // from the [`Interpreter`]; however, if the interpreter is a virtual environment itself, then
     // the base Python executable is the Python executable of the interpreter's base interpreter.
+    let base_executable = interpreter
+        .sys_base_executable()
+        .unwrap_or(interpreter.sys_executable());
     let base_python = if cfg!(unix) && interpreter.is_standalone() {
```

Comment:
Is it still correct to only gate this to standalone interpreters?

Reply:
It's possible that this would be equally applicable to other interpreters, but I'm a little more hesitant... The `else` behavior matches the standard library, and we're just less familiar with how those "other interpreters" behave.

Reply:
Okay. Fine with me, this logic seems very "correct" though.